GCS_util.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package source
+package connector
```

Comment:
Why not call it source_test?

Reply:
I think we have one directory and one package in Golang.
In the parent directory we are using connector package so iam using the same for util and test files too.